infrastructure/terraform/README.md

```diff
@@ -78,25 +78,15 @@ installation.
     vim ${TERRAFORM_RUN_DIR}/terraform.tfvars
     ```
 
-6. Run Terraform to create resource for **marketing data store** (the Dataform repository and related workflows):
+6. Run Terraform to create resources for **marketing data engine**:
```

Comment:
I would drop "for marketing data engine" altogether - we don't need to differentiate any more.

Reply:
updated